lib/modules/ens/embarkjs.js

```diff
@@ -285,11 +172,13 @@ __embarkENS.resolve = function(name) {
   }).catch(err => err);
 };
 
-__embarkENS.lookup = function(address) {
+__embarkENS.lookup = function (address, cb) {
```

Comment:
why was a callback added? this was returning a promise afak

Reply:
I asked for it, because the final values (name) were never "returned". Seems like it only returns the promise of the resolver, which is not what we want in the end (I think)

Reply:
@jrainville suggested I use it.